report/src/main/kotlin/com/hartwig/actin/report/pdf/tables/trial/ActinTrialGeneratorFunctions.kt

```diff
@@ -68,11 +68,14 @@ object ActinTrialGeneratorFunctions {
                 cohort.phase?.takeIf { it != TrialPhase.COMPASSIONATE_USE }
                     ?.let { Text("\n(${it.display()})").addStyle(Styles.tableContentStyle()) })
 
+
             table.addCell(
                 when (cohort.source) {
-                    TrialSource.LKO -> createContent(Paragraph().addAll(trialLabelText.map { it.addStyle(Styles.urlStyle()) })).setAction(
-                        PdfAction.createURI(cohort.trialId.replace("LKO", "https://longkankeronderzoek.nl/studies/"))
-                    )
+                    TrialSource.LKO -> cohort.sourceId?.let {
+                        createContent(Paragraph().addAll(trialLabelText.map { it.addStyle(Styles.urlStyle()) })).setAction(
+                            PdfAction.createURI(cohort.sourceId.replace("LKO", "https://longkankeronderzoek.nl/studies/"))
```

Comment:
Should we continue to depend on prefixing "LKO" to the ids? I would expect that in the trial we would just have sourceId = "123" and source = "LKO", then this code would be "https://longkankeronderzoek.nl/studies/{sourceId}".